packages/js/product-editor/src/components/attribute-input-field/attribute-input-field.tsx

```diff
@@ -200,7 +221,12 @@ export const AttributeInputField: React.FC< AttributeInputFieldProps > = ( {
 											</span>
 										</div>
 									) : (
-										item.name
+										<>
+											{ item.name }
+											{ item.isDisabled &&
+												disabledAttributeMessage &&
+												` ${ disabledAttributeMessage }` }
```

Comment:
Is there a reason you end up going with this over the Tooltip? Maybe this is something you had discussed with @jarekmorawski ?

Reply:
<img width="337" alt="Screenshot 2023-08-17 at 10 06 16 AM" src="https://github.com/woocommerce/woocommerce/assets/2240960/3cceaba4-23d4-45ab-9b2a-2dceb30e400e">

Reply:
Maikel raised this on Slack (p1692201837100299-slack-C01CMSA1VHS) saying that there are technical constraints. Ideally, we'd use a tooltip, but if that isn't possible, maybe we could show the message as the item's description? If that isn't possible either, it's best to not show affected attributes at all (the message show in-line with the attribute's name would likely cause more harm than good).

Reply:
On a different note, can we do anything to make the dropdown list the same width as the field? It looks a bit off now.